src/zabapgit_object_clas.prog.abap

```diff
@@ -200,13 +200,22 @@ CLASS lcl_object_clas IMPLEMENTATION.
           lt_descriptions TYPE lif_defs=>ty_seocompotx_tt,
           ls_clskey       TYPE seoclskey,
           lt_sotr         TYPE lif_defs=>ty_sotr_tt,
-          lt_lines        TYPE tlinetab.
-
+          lt_lines        TYPE tlinetab,
+          lv_curr_lang    TYPE langu,
+          lv_dummy        TYPE string ##needed.
 
     ls_clskey-clsname = ms_item-obj_name.
 
+    "If class was deserialized with a previous versions of abapGit and current language was different
+    "from master language at this time, this call would return SY-LANGU as master language. To fix
+    "these objects, set SY-LANGU to master language temporarily.
+    GET LOCALE LANGUAGE lv_curr_lang COUNTRY lv_dummy MODIFIER lv_dummy.
+    SET LOCALE LANGUAGE mv_language.
```

Comment:
Thomas,

Thanks for your pull request 👍 .

I notice that this is duplicated three times on the pull request. It is possible to extract this into a class? See [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Reply:
That's a good point. Maybe creating a class could make sense here, because I expect this issue to appear for other calls to SAP function modules as well.